src/common/CommonInterfaces.ts

```diff
@@ -58,7 +58,7 @@ export interface ITransaction {
         blockHash: string | null,
         blockNumber: number,
         from: string,
-        gas: string,
+        gas: number,
```

Comment:
Does this changes return value?

Reply:
Initial `gas` type was defined wrong. It does not return value, it used as an interface.